lib/airbrake.js

```diff
@@ -183,11 +186,17 @@ Airbrake.prototype.notify = function (err, cb) {
     }
   });
 
-  if (exit) {
+  var notice = this.notifyJSON(err);
+
+  this.filters.forEach(function (filter) {
+    notice = filter(notice);
```

Comment:
notice is modified here intentionally, so that it can be modified with custom information in a filter if deemed necessary. For example:

```
airbrake.addFilter(function(notice) {
  notice.context.environment = 'production';
  notice.context.version = '1.2.3';
  return notice;
});
```

I have added an additional check that future filters will not be ran if the notice is null. This should result in expected and useful behavior.